dataflow/src/main/java/org/checkerframework/dataflow/analysis/BackwardTransferFunction.java

```diff
@@ -0,0 +1,53 @@
+package org.checkerframework.dataflow.analysis;
+
+import java.util.List;
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.checkerframework.dataflow.cfg.UnderlyingAST;
+import org.checkerframework.dataflow.cfg.node.Node;
+import org.checkerframework.dataflow.cfg.node.ReturnNode;
+
+/**
+ * Interface of a backward transfer function for the abstract interpretation used for the backward
+ * flow analysis.
+ *
+ * <p>A backward transfer function consists of the following components:
+ *
+ * <ul>
+ *   <li>A method {@code initialNormalExitStore} that determines which initial store should be used
+ *       at the normal exit block in the org.checkerframework.dataflow backward analysis.
```

Comment:
Is the `org.checkerframework.dataflow` part really adding anything useful in these three bullet points?